test/Snapshot/InMemorySnapshotRepositoryTest.php

```diff
@@ -13,6 +13,31 @@
 class InMemorySnapshotRepositoryTest extends SnapshotRepositoryTest
 {
     /**
+     * @test
+     */
+    public function it_does_not_mutate_state_of_Snapshot_Aggregate()
+    {
+        $aggregate = $this->createAggregateWithHistory(5);
+        $this->repository->save(new Snapshot($aggregate));
```

Comment:
this is already tested in SnapshotRepositoryTest::it_returns_snapshot_when_available